### DISCUSSION TITLE
Support for adding traces/observations/sessions/events/spans/generations related logging to LF via API

### DISCUSSION THREAD

**Discussion by ashutoshsaboo** [langfuse/langfuse#2705] [Support] [answered]
### Describe the feature or potential improvement

Same as title. Tbh a bit surprised to see that this support is not vended out via the REST API (https://api.reference.langfuse.com/) - especially given that this is one of the most core features of Langfuse. Can this support be please added to the REST API?

**Comment by marcklingen** [comment, marked as answer]
All APIs used by the Langfuse SDKs are public, tracing data is sent via the batched /ingestion endpoint for performance reasons. Many teams implement their own client implementations for languages/frameworks that are not supported by one of the official clients.

Have a look here: https://api.reference.langfuse.com/#post-/api/public/ingestion

Let me know in case you have any questions on this.

**Reply by ashutoshsaboo** [reply]
Ahh I missed seeing that this api already supports it. My bad, and thank you for answering, Marc. I'll reply on this if I face any issues with pushing traces via this.

**Reply by marcklingen** [reply]
no worries, great to have this here as an FAQ item
